src/patternfly/components/TreeView/examples/TreeView.md

```diff
@@ -0,0 +1,840 @@
+            {{/tree-view-list}}
+        {{/tree-view-list-item}}
+        {{#> tree-view-list-item tree-view-list-item--expandable="true"}}
+            {{#> tree-view-link}}
+                {{> tree-view-link-toggle}}
+                {{#> tree-view-link-text}}
+                    Cost management
+                {{/tree-view-link-text}}
+            {{/tree-view-link}}
+        {{/tree-view-list-item}}
+        {{#> tree-view-list-item tree-view-list-item--expandable="true"}}
+            {{#> tree-view-link}}
+                {{> tree-view-link-toggle}}
+                {{#> tree-view-link-text}}
+                    Sources
+                {{/tree-view-link-text}}
+            {{/tree-view-link}}
+        {{/tree-view-list-item}}
+        {{#> tree-view-list-item tree-view-list-item--expandable="true"}}
+            {{#> tree-view-link-wrapper}}
+                {{#> tree-view-link}}
+                    {{> tree-view-link-toggle}}
+                    {{#> tree-view-link-text}}
+                        This is a really really really long folder name that overflows from the width of the container.
+                    {{/tree-view-link-text}}
+                {{/tree-view-link}}
+                {{#> tree-view-action tree-view-action--attribute='aria-label="Alert"'}}
+                    <i class="fas fa-bell" aria-hidden="true"></i>
+                {{/tree-view-action}}
+            {{/tree-view-link-wrapper}}
+        {{/tree-view-list-item}}
+    {{/tree-view-list}}
+{{/tree-view}}
+```
+
+## Documentation
+### Overview
+
+### Accessibility
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `role="tree"` | `.pf-c-tree-view__list` | Identifies the `ul` as a tree widget. **Place on the first ul only** |
+| `role="group"` | `.pf-c-tree-view__list` | Identifies the `ul` element as a container of treeitem elements that form a branch of the tree. **Place on all ul's except the first ul** |
+| `role="treeitem"`| `.pf-c-tree-view__link` | Identifies the link as a tree item. **Note:** The role is set on the element that is interactive and focusable. |
+| `role="none"` | `.pf-c-tree-view__list-item` | Hides the implicit listitem role of the li element from assistive technologies. |
+| `aria-expanded="false"` | `.pf-c-tree-view__list-item.pf-m-expandable` | Indicates the parent node is closed, i.e., the descendant elements are not visible. |
+| `aria-expanded="true"` | `.pf-c-tree-view__list-item.pf-m-expandable.pf-m-expanded` | Indicates the parent node is open, i.e., the descendant elements are visible. |
+| `tabindex="-1"` | `.pf-c-tree-view__link` | Makes the element with the treeitem role focusable without including it in the tab sequence of the page. |
+| `tabindex="0"` | `.pf-c-tree-view__link` | Includes the element with the treeitem role in the tab sequence. Only one treeitem in the tree has tabindex="0". When the user moves focus in the tree, the element included in the tab sequence changes to the element with focus. |
```

Comment:
how do you navigate the tree by keyboard? All of our other menus like this (like nav) allow you to tab/focus on all of the items.

Reply:
This is what I was following for the tree view accessibility https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-2/treeview-2a.html 

- "All treeitem elements are focusable, but only one is included in the tab sequence."
- "Only one treeitem in the tree has tabindex="0"."

Reply:
Ah gotcha, the keyboard navigation is managed via JS.